AliceO2Group/RunConditionTable
# rctmake rebuild option

#### I DON'T have JIRA ticket
- [x] explain what this PR does
- [ ] if it is a new feature, explain how you plan to use it
- [ ] tests are added
- [x] documentation was updated or added

Notable changes for users (see rctmake help changes):
- add new feature to rctmake
- change syntax for stage run
- prune renamed to rm
- add stage prune
